kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -185,6 +185,10 @@ private String getRowId(SinkRecord record) {
   @Override
   public void put(Collection<SinkRecord> records) {
     logger.info("Putting {} records in the sink.", records.size());
```

Comment:
Nit: might want to move this log statement after the error check, since we won't actually be putting anything in the sink if we're about to fail the task.